flutter/lib/src/view_hierarchy/sentry_tree_walker.dart

```diff
@@ -54,13 +54,9 @@ class _TreeWalker {
 
     final renderObject = element.renderObject;
     if (renderObject is RenderBox) {
-      final offset = renderObject.localToGlobal(Offset.zero);
-      if (offset.dx > 0) {
-        x = offset.dx;
-      }
-      if (offset.dy > 0) {
-        y = offset.dy;
-      }
+      final offset = renderObject.globalToLocal(Offset.zero);
+      x = offset.dx;
+      y = offset.dy;
```

Comment:
@narsaynorath I've noticed negative values for some reason, is this normal?
I can filter out negative and zero values if needed, let me know.